### ISSUE TITLE
linux-esx is missing a bunch of cgroup compilation params

### ISSUE THREAD

**Issue by casualjim** [vmware/photon#487]
when I run docker on the default linux kernel, i get no warnings in my logs. as soon as I install linux-esx package I see them. Most of them are compilation params for the kernel, so I'm wondering is there a reason they are not switched on?

In these logs you first see the startup of docker on default linux kernel, there  is a bit that says -- reboot -- this reboot means I did `tdnf install -y linux-esx`

```
-- Logs begin at Thu 2016-07-07 16:06:14 UTC, end at Thu 2016-07-07 16:09:00 UTC. --
Jul 07 16:06:19 Photon systemd[1]: Starting Docker Application Container Engine...
Jul 07 16:06:20 Photon docker[718]: time="2016-07-07T16:06:20.447997521Z" level=warning msg="/!\\ DON'T BIND ON ANY IP ADDRESS
WITHOUT setting -tlsverify IF YOU DON'T KNOW WHAT YOU'RE DOING /!\\"
Jul 07 16:06:20 Photon docker[718]: time="2016-07-07T16:06:20.716892566Z" level=info msg="Graph migration to content-addressabi
lity took 0.00 seconds"
Jul 07 16:06:20 Photon docker[718]: time="2016-07-07T16:06:20.802648758Z" level=info msg="Firewalld running: false"
Jul 07 16:06:20 Photon docker[718]: time="2016-07-07T16:06:20.992566298Z" level=info msg="Default bridge (docker0) is assigned
with an IP address 172.17.0.0/16. Daemon option --bip can be used to set a preferred IP address"
Jul 07 16:06:21 Photon docker[718]: time="2016-07-07T16:06:21.065947516Z" level=info msg="Loading containers: start."
Jul 07 16:06:21 Photon docker[718]: time="2016-07-07T16:06:21.066295683Z" level=info msg="Loading containers: done."
Jul 07 16:06:21 Photon docker[718]: time="2016-07-07T16:06:21.066309627Z" level=info msg="Daemon has completed initialization"
Jul 07 16:06:21 Photon docker[718]: time="2016-07-07T16:06:21.066320187Z" level=info msg="Docker daemon" commit=4dc5990 graphdr
iver=overlay version=1.11.0
Jul 07 16:06:21 Photon systemd[1]: Started Docker Application Container Engine.
Jul 07 16:06:21 Photon docker[718]: time="2016-07-07T16:06:21.072327204Z" level=info msg="API listen on [::]:2376"
Jul 07 16:06:21 Photon docker[718]: time="2016-07-07T16:06:21.072391879Z" level=info msg="API listen on /var/run/docker.sock"
Jul 07 16:08:31 Photon systemd[1]: Stopping Docker Application Container Engine...
Jul 07 16:08:31 Photon systemd[1]: Stopped Docker Application Container Engine.
Jul 07 16:08:31 Photon docker[718]: time="2016-07-07T16:08:31.823111426Z" level=info msg="Processing signal 'terminated'"
-- Reboot --
Jul 07 16:08:50 Photon systemd[1]: Starting Docker Application Container Engine...
Jul 07 16:08:51 Photon docker[270]: time="2016-07-07T16:08:51.693523188Z" level=warning msg="/!\\ DON'T BIND ON ANY IP ADDRESS
WITHOUT setting -tlsverify IF YOU DON'T KNOW WHAT YOU'RE DOING /!\\"
Jul 07 16:08:51 Photon docker[270]: time="2016-07-07T16:08:51.819059919Z" level=info msg="Graph migration to content-addressabi
lity took 0.00 seconds"
Jul 07 16:08:51 Photon docker[270]: time="2016-07-07T16:08:51.842387057Z" level=info msg="Firewalld running: false"
Jul 07 16:08:51 Photon docker[270]: time="2016-07-07T16:08:51.940015798Z" level=info msg="Default bridge (docker0) is assigned
with an IP address 172.17.0.0/16. Daemon option --bip can be used to set a preferred IP address"
Jul 07 16:08:51 Photon docker[270]: time="2016-07-07T16:08:51.981958613Z" level=warning msg="Your kernel does not support kerne
l memory limit."
Jul 07 16:08:51 Photon docker[270]: time="2016-07-07T16:08:51.982141427Z" level=warning msg="Your kernel does not support cgrou
p blkio weight"
Jul 07 16:08:51 Photon docker[270]: time="2016-07-07T16:08:51.982268825Z" level=warning msg="Your kernel does not support cgrou
p blkio weight_device"
Jul 07 16:08:51 Photon docker[270]: time="2016-07-07T16:08:51.982401114Z" level=warning msg="Your kernel does not support cgrou
p blkio throttle.read_bps_device"
Jul 07 16:08:51 Photon docker[270]: time="2016-07-07T16:08:51.982518107Z" level=warning msg="Your kernel does not support cgrou
p blkio throttle.write_bps_device"
Jul 07 16:08:51 Photon docker[270]: time="2016-07-07T16:08:51.982630579Z" level=warning msg="Your kernel does not support cgrou
p blkio throttle.read_iops_device"
Jul 07 16:08:51 Photon docker[270]: time="2016-07-07T16:08:51.982737682Z" level=warning msg="Your kernel does not support cgrou
p blkio throttle.write_iops_device"
Jul 07 16:08:51 Photon docker[270]: time="2016-07-07T16:08:51.982937961Z" level=warning msg="mountpoint for pids not found"
Jul 07 16:08:51 Photon docker[270]: time="2016-07-07T16:08:51.984639644Z" level=info msg="Loading containers: start."
Jul 07 16:08:51 Photon docker[270]: time="2016-07-07T16:08:51.985038969Z" level=info msg="Loading containers: done."
Jul 07 16:08:51 Photon docker[270]: time="2016-07-07T16:08:51.985177898Z" level=info msg="Daemon has completed initialization"
Jul 07 16:08:51 Photon docker[270]: time="2016-07-07T16:08:51.985290191Z" level=info msg="Docker daemon" commit=4dc5990 graphdr
iver=overlay version=1.11.0
Jul 07 16:08:51 Photon systemd[1]: Started Docker Application Container Engine.
Jul 07 16:08:51 Photon docker[270]: time="2016-07-07T16:08:51.990932920Z" level=info msg="API listen on /var/run/docker.sock"
Jul 07 16:08:51 Photon docker[270]: time="2016-07-07T16:08:51.990905874Z" level=info msg="API listen on [::]:2376"
```

**Comment by YustasSwamp** [comment]
Hi Ivan,
It will be fixed in linux-esx-4.4.8-8.
But messages for missing blkio.{weight,weight_device} will be still there. These features are part of CFQ IO scheduler. It's disabled in linux-esx.

**Comment by casualjim** [comment]
ok thanks! I mainly care about ram limits anyway
